### PR TITLE
French translations for missing parameters (access control and batch loading)

### PR DESCRIPTION
**Pull request by pilasou**
## Description
Addition to the json5 french file to fix missing parameters for access control and batch loading in DSpace.

### CODE REVIEW

File: src/assets/i18n/fr.json5 [hunk 1]
```diff
@@ -313,6 +313,33 @@
...
+  //"admin.access-control.bulk-access-browse.search.header": "Search",
+  "admin.access-control.bulk-access-browse.search.header": "Chercher",
+
+
+  //"admin.access-control.bulk-access-browse.selected.header": "Current selection({{number}})",
+  "admin.access-control.bulk-access-browse.selected.header": "Sélection actuelle({{number}})",
+
+
+  //"admin.access-control.bulk-access-settings.header": "Step 2: Operation to Perform",
+  "admin.access-control.bulk-access-settings.header": "Étape 2 : Opération à éxecuter",
```
Comment (mhvezina) [inline src/assets/i18n/fr.json5:341]: Should be "Étape 2 : Opération à exécuter"

File: src/assets/i18n/fr.json5 [hunk 2]
```diff
@@ -772,18 +799,34 @@
...
   "admin.metadata-import.page.header": "Importer Métadonnées",
 
+  //"admin.batch-import.page.header": "Import Batch",
+  "admin.batch-import.page.header": "Importation en lots",
+
   // "admin.metadata-import.page.help": "You can drop or browse CSV files that contain batch metadata operations on files here",
   "admin.metadata-import.page.help": "Vous pouvez glisser-déposer ou rechercher ici des fichiers CSV qui contiennent des opérations en lot sur les métadonnées",
 
+//"admin.batch-import.page.help": "Select the Collection to import into. Then, drop or browse to a Simple Archive Format (SAF) zip file that includes the Items to import",
+"admin.batch-import.page.help": "Sélectionner la collection dans laquelle importer les données. Ensuite, ajouter le fichier le fichier zip Simple Archive Format (SAF) contenant tous les Items à importer.",
```
Comment (mhvezina) [inline src/assets/i18n/fr.json5:818]: Should be : "Sélectionner la collection dans laquelle importer les données. Ensuite, ajouter le fichier zip Simple Archive Format (SAF) contenant tous les Items à importer."

File: src/assets/i18n/fr.json5 [hunk 3]
```diff
@@ -6251,6 +6310,63 @@
...
+"access-control-replace-all": "Remplacer les conditions d'accès",
+
+//"access-control-add-to-existing":"Add to existing ones", 
+"access-control-add-to-existing":"Ajouter aux conditions d'accès existantes", 
+
+//"access-control-limit-to-specific": "Limit the changes to specific bitstreams",
+"access-control-limit-to-specific": "Limiter les modifications à des Bitstreams spécifiques",
+
+//"access-control-process-all-bitstreams": "Update all the bitstreams in the item",
+"access-control-process-all-bitstreams": "Mise à jour de tous les Bitstreams de l'item",
```
Comment (mhvezina) [inline src/assets/i18n/fr.json5:6359]: Should be "Mise à jour de tous les Bitstreams de l'Item"

**Comment by mhvezina** [comment]
Thank you @pilasou 

I added a few suggestions to the translations. @pilasou : you might want to correct your title issue "French rranslations..."

@tdonohue everything else seems good to me!

**Review by mhvezina — COMMENTED** [review]
https://github.com/DSpace/dspace-angular/pull/2831#issuecomment-1961649466

File: src/assets/i18n/fr.json5 (hunk 1 above)
Comment (pilasou) [reply src/assets/i18n/fr.json5]: Done in last commit. 46df7e805e2eac71c31acd35bd15367ceb5efc46

File: src/assets/i18n/fr.json5 (hunk 2 above)
Comment (pilasou) [reply src/assets/i18n/fr.json5]: Done in last commit. 46df7e805e2eac71c31acd35bd15367ceb5efc46

File: src/assets/i18n/fr.json5 (hunk 3 above)
Comment (pilasou) [reply src/assets/i18n/fr.json5]: Done in last commit. 46df7e805e2eac71c31acd35bd15367ceb5efc46

**Comment by tdonohue** [comment]
@pilasou : This PR appears to be failing `lint` checks.  Could you update it to fix the small issues?   They should be visible to you locally by running `yarn lint`, and you might be able to run `yarn lint --fix` to fix them automatically.  (The automatic fixes don't always work)

**Comment by tdonohue** [comment]
@pilasou : There's another lint formatting error now.  Here's the new one:

```
/home/runner/work/dspace-angular/dspace-angular/src/assets/i18n/fr.json5
Lint errors found in the listed files.

  6635:38  error  Parsing error: Unexpected character '“'
```

It appears to think that one of the double-quotes (") on line 6635 is invalid?  Maybe it's accidentally a special character and not a double-quote character?  It's unclear to me though because the error isn't obvious when you view this file in GitHub.  (My suspicion is maybe this text was copied from a different document like Word and it changed the character)

**Comment by tdonohue** [comment]
I've pushed up some additional lint fixes as this still had lint failures.  I simply checked out this PR and ran `yarn lint --fix` to automatically apply the fixes.  Once the build succeeds, I'll get this merged & backported to 7.x

**Review by tdonohue — APPROVED** [review]
👍 Tested locally and it works now.  Lint also passes

**Pull request merged**